src/pycsou/abc/operator.py

```diff
@@ -1752,11 +1755,11 @@ def lipschitz(self, recompute: bool = False, algo: str = "svds", gpu: bool = Fal
             Algorithm used for computing the Lipschitz constant. If ``algo==svds`` the Lispchtiz constant is estimated as the
             spectral norm of :math:`L`, computed via Scipy's :py:func:`scipy.sparse.linalg.svds` routine (more accurate but more compute intensive).  If ``algo==fro`` the Lispchtiz constant is estimated as the
             Froebenius norm of :math:`L`, computed via the matrix-free `Hutch++ stochastic trace estimation algorithm <https://arxiv.org/abs/2010.09649>`_ (less accurate but less compute intensive).
-            Currently, only ``algo==svds`` is supported.
+
         kwargs:
             Optional arguments to be passed to the algorithm used for computing the Lipschitz constant.
             The parameter ``tol`` of Scipy's :py:func:`scipy.sparse.linalg.svds` routine can be particularly interesting to reduce
-            the compute time of the Lipschitz constant.
+            the compute time of the Lipschitz constant. The parameter ``m`` of LinOps :py:func:`pycsou.abc.operator.LinOp.trace` can also be interesting to control the number of queries used for the stochastic estimation.
```

Comment:
Make it clearer which algorithm is affected by the optional keywords``tol`` or ``m``.